### PR TITLE
:bug: Fixed code in order to make PAM work in coco after health check…

### PR DESCRIPTION
**Pull request by izzyblues**
… library upgrade

### CODE REVIEW

File: app.go [hunk 1]
```diff
@@ -138,22 +138,10 @@ func main() {
 
 	if *etcdPeers == "NOT_AVAILABLE" {
 		log.Info("Sourcing dynamic configs from file")
-		err = updateEnvsIfChanged(*envsFileName, *envCredentialsFileName)
```
Comment (tamas-molnar) [inline app.go]: Why was this part removed? It helps quickly diagnose missing/invalid configs on startup.

Comment (khatton-ft) [reply app.go]: Furthermore, we actually need the config to be loaded before the healthchecks are initialised, otherwise the service remains unhealthy because the healthcheck believes there are no read environments. I've addressed this in the commit below by executing the code immediately before going into the timer's 1-minute loop.

**Comment by izzyblues** [comment]
👍
